code/game/machinery/kitchen/processor.dm

```diff
@@ -47,6 +47,14 @@
 		input = /obj/item/reagent_container/food/snacks/flour
 		output = /obj/item/reagent_container/food/snacks/spagetti
 
+	spaghetti
```

Comment:
Hate to tell you but this ain't spaghetti

Reply:
corrected